SPECS/xen.spec

```diff
@@ -104,6 +105,7 @@ Patch3010: xsa180-qemut.patch
 Patch3011: xsa184-qemut-master.patch
 Patch3012: xsa197-qemut.patch
 Patch3013: xsa199-trad.patch
+Patch3014: xsa208-qemut.patch
 
```

Comment:
Hey Johnny,

Just so you know my intent here was to keep the patch number offsets synced; i.e., if xsa208 for qemut has patch 3014, xsa208 for qemuu should be 2014 as well (even if we end up skipping a patch).  I'll fix that up later though.